components/net/http_loader.rs

```diff
@@ -1366,14 +1383,20 @@ fn has_credentials(url: &ServoUrl) -> bool {
 }
 
 fn is_no_store_cache(headers: &Headers) -> bool {
+    println!("debug headers {:?}", headers);
```

Comment:
@KiChjang In the context of the `http-cache/cc-request.html` tests, it seems that the request headers set in the tests don't come through to the headers in the request inside `network_or_cache_fetch`. It seems that some basic headers are set, but not the ones the test is adding. Could this be a bug in the tests, or is some part of the fetch code dealing with request headers not implemented yet?

```
 "debug headers Headers { Accept: */*"
", Accept-Language: en-US, en; q=0.5"
", Referer: http://web-platform.test:8000/fetch/http-cache/cc-request.html"
", User-Agent: Servo/wptrunner"
", }"
```

Reply:
https://dxr.mozilla.org/servo/rev/04f787dbf9b235bf3af58dc7f3160689cdb2311c/components/script/dom/request.rs#292-343 is the code that is supposed to propagate the headers specified in the test into the network stack.

Reply:
thanks, I'll check it out...

Reply:
fixed in https://github.com/servo/servo/pull/18676/commits/f4f7b9f54f0c854f468cd37eae967f26798789e6

The headers were not being copied from the dom Request to the net Request...